tests/nlp/utils/test_properties.py

```diff
@@ -182,6 +182,25 @@ def test_calculate_average_sentence_length_property(tweet_emotion_train_test_tex
     assert_that(result_none_text['Average Words Per Sentence'], equal_to([np.nan]))
 
 
+def test_batch_size_change(tweet_emotion_train_test_textdata):
+    # Arrange
+    _, test = tweet_emotion_train_test_textdata
+    test_text = test.text
+    batch_sizes=[1, 8, 16, 32, 64]
+
+    # Act
+    for batch in batch_sizes:
+        result = calculate_builtin_properties(test_text, include_properties=['Average Words Per Sentence'],
+                                              batch_size=batch)[0]
+        result_none_text = calculate_builtin_properties([None], include_properties=['Average Words Per Sentence'],
+                                                        batch_size=batch)[0]
+
+        # Assert
+        assert_that(result['Average Words Per Sentence'][0: 10], equal_to([5.667, 7.0, 11.0, 12.0, 8.0, 19.0, 3.0, 9.0,
```

Comment:
add assertion also on property that is english only to test if works well also there